### ISSUE TITLE
How to deal with subject emphasis with ол

### ISSUE THREAD

**Issue by jonorthwash** [ftyers/docs#7]
From the discussion started in [issue 6](https://github.com/ftyers/docs/issues/6):
In terms of "Біз ол гол соққан", can you say "Біз ол гол соқтық" or "Біз ол гол соғамыз" (using the same construction, not in the meaning of using ол as a determiner for гол)?

My intuition is that this is more like an appositive construction of some sort, so you need something attributive or substantive (e.g., a verbal noun or adjective, like соққан, соғатын, or соғар—or just a noun or adjective, like өтірікші), but you can't have a finite verb (like соқтық, соғамыз, etc.). (I would also opt to put a comma after ол in all of these examples. It seems like it could get confusing without a comma since ол can also be a determiner, but I'm not familiar with the relevant standard in Kazakh.)

Anyway, I'm not a native speaker of Kazakh, so my intuitions on this could very well be wrong. What do you think?

**Comment by makazhan** [comment]
You are correct, finite verbs cannot be used in this construction. Regarding the comma, I am pretty sure it is usually not present. Right now, I cannot confirm an appositive construction intuition, but as I see it, a noun (nominal) phrase preceding "ол" is the subject of whatever follows "ол". So, to me, "ол" here functions as a copula.

**Comment by jonorthwash** [comment]
So are you saying that "Біз" is the subject of "гол соққан" in "Біз ол гол соққан"?  If so, why isn't there agreement morphology (i.e., why is it not "соққанбыз")?

**Comment by makazhan** [comment]
I agree there should be agreement. It's just semantically _"сен өтірікшісің"_ and _"сен ол өтірікші"_ sound the same to me with the exception of the emphasis inflected by _"ол"_.

On a `NP ол VP/Adj` example, what are possible analysis options?
- Can we say that `VP/Adj` is the predicate?
- Can we say that `nsubj(NP,ол)` which is in turn is `csubj` for `VP/Adj`?
- Can we say that `nsubj(NP,ол)` and `appos(VP/Adj,ол)` or `appos(ол,VP/Adj)`?

*everywhere the relation order is `(dep,gov)`

**Comment by jonorthwash** [comment]
What about `appos(ол, NP)`?

I'm open to various possibilities, but my main motivations for how to analyse this construction will be the answers to the following questions, which I think we haven't investigated thoroughly enough yet:
- What are the possible forms for what you're calling "`V/Adj`"?  Nouns are okay ("өтірікші", unless you consider this an adjective?).  Verbal adjectives ("соққан", maybe substantivised?  or perhaps these are actually verbal nouns??) seem to be allowed.  I think I found an examples in google of pronouns being used ("Мен ол мен, сен ол сен") too.  So maybe we need clearer examples of nouns (and bigger noun phrases?), adjectives, verbal adjectives (with bigger verb phrases), and verbal nouns (also with bigger verb phrases), or whichever of those are possible.  And we know it can't be a finite verb.
- Whatever the restrictions on the ~predicate element, it "agrees" as 3rd person (assuming it's not just some fragment that doesn't agree at all?).  So having an `nsubj` relation between `NP` and it seems bad, since there's not an agreement relation between them.  Related to this, can you ever say something like "Мен олмын" or "Сен олсың ба?" or "Сен ол бе едің" or "Ол мен емес"?  If so, what would they mean / what contexts would you use say something like these in?
- However we decide to analyse it, we should acknowledge the existence of a copula.  Could there be a null copula at the end, just like in the simple "Ол өтірікші"?
- If there is a null copula, can it ever be made overt?  E.g., what happens if we try to get a past-tense reading?  "Сен ол өтірікші еді"?  What about some other tense, like future ("болады"??)?
